helm/default-apps-cloud-director/values.yaml

```diff
@@ -156,6 +156,21 @@ apps:
     # used by renovate
     # repo: giantswarm/observability-bundle
     version: 0.8.9
+  teleportKubeAgent:
+    appName: teleport-kube-agent
+    chartName: teleport-kube-agent
+    catalog: default
+    clusterValues:
+      configMap: true
+      secret: true
+    forceUpgrade: true
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/teleport-kube-agent-app
+    version: 0.6.0
+    extraConfigs:
+    - kind: configMap
+      name: "{{ $.Values.clusterName }}-teleport-kube-agent-config"
```

Comment:
I think this is a recurring question. Maybe adding a comment that this is created with `giantswarm/teleport-operator` would be helpful